settings.example.js

```diff
@@ -1,7 +1,16 @@
 var settings = {
-    username: "develina.devsson@mydevtenant.onmicrosoft.com",
-    password: "pass@word1",
-    siteUrl: "https://mydevtenant.sharepoint.com/" 
+
+    spsave: {
+        username: "develina.devsson@mydevtenant.onmicrosoft.com",
+        password: "pass@word1",
+        siteUrl: "https://mydevtenant.sharepoint.com/"
+    },
+    testing: {
+        clientId: "{ client id }",
```

Comment:
@patrick-rodgers, i was reading PR #152 and i know that i need to register my app in ***appregnew.aspx***, but,  i have a question, why register in ***appinv.aspx***?
I must register in both places?***(appregnew.aspx and appinv.aspx)***

![image](https://cloud.githubusercontent.com/assets/2263551/16886506/2a13f508-4aab-11e6-97b8-35b09e7f49b7.png)

Thanks!